src/test/unit/spec1.js

```diff
@@ -0,0 +1,26 @@
+describe("app", function () {
+
+  var $rootScope;
+  var $controller;
+
+  beforeEach(module("openevent"));
+  beforeEach(inject(function ($injector) {
+
+    $rootScope = $injector.get("$rootScope");
+    $controller = $injector.get("$controller");
+    $scope = $rootScope.$new();
+
+  }));
+  beforeEach(inject(function ($controller) {
+    AppController = $controller("AppController");
+
+  }));
+
+    it("It will describe the title variable", function () {
+
+    expect(AppController.appTitle).toBe(config.title);
+    expect(openevent.totalDays).toBe(0);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["expect" is not defined.](https://www.codacy.com/app/triplez/open-event-webapp/file/2533664929/issues/source?bid=3233857&fileBranchId=3284028#l22)